src/scribe_data/wikidata/format_data.py

```diff
@@ -51,16 +51,18 @@ def format_data(
 
```

Comment:
Apart from the changes already in the PR, should the changes also include modifying that new workflow that's being worked on, [`check_and_update_missing_query_forms.yaml`](https://github.com/scribe-org/Scribe-Data/blob/main/.github/workflows/check_and_update_missing_query_forms.yaml)? To also include the additional segments for `lastModified` in the generated queries?

Reply:
True yes :)

Reply:
We also need the gender concatenations for #564 to be included in the query generations 🤔

Reply:
But then I worry a bit about putting group by into all of the queries like this. Maybe the thing to do is handle this in the formatting script where basically values are concatenated if they're already present in the resulting data. Spanish nouns gender just isn't really a useful field with the way the data has been structured.

And also, I just remembered that @axif0 mentioned that this has been added to the other PR in https://github.com/scribe-org/Scribe-Data/pull/563/commits/e363282c9c8833b57f3bdaa7373ed09b9636c8a5 😊

Reply:
So we're good to go here :) :)

Reply:
> And also, I just remembered that @axif0 mentioned that this has been added to the other PR

Ah nice! 
I hadn't seen that before - good stuff! :+1: way to go